crates/sui-source-validation/src/tests.rs

```diff
@@ -567,16 +606,10 @@ async fn copy_package<'s>(
     tokio::fs::create_dir(&dst).await?;
 
     // Copy TOML
-    let dst_toml = dst.join("Move.toml");
-    tokio::fs::copy(src.join("Move.toml"), &dst_toml).await?;
-
-    {
-        let mut toml = fs::OpenOptions::new().append(true).open(dst_toml)?;
-        writeln!(toml, "[addresses]")?;
-        for (name, addr) in addresses {
-            writeln!(toml, "{name} = \"{addr}\"")?;
-        }
-    }
+    let mut toml = tokio::fs::read_to_string(src.join("Move.toml")).await?;
+    toml = toml.replace("$REPO_ROOT", &repo_root.to_string_lossy());
```

Comment:
The new test needs to be able to depend on the sui-framework, in the repo root, but we can't use a local dependency, because fixtures are moved into a temporary directory to update their manifests, so I'm using a bit of a hacky templating/variable substitution system here.